spec/controllers/registered_applications_controller_spec.rb

```diff
@@ -35,6 +35,8 @@
         get :show, params: {id: my_app.id}
         expect(assigns(:registered_application)).to eq(my_app)
       end
+      
```

Comment:
The point of that line is that in the RegisteredApplicationsController, there is a line:

```
def show
         @registered_application = RegisteredApplication.find(params[:id])
...
```

This line is tested by writing
```
expect(assigns(:registered_application)).to eq(my_app)
```

where `my_app` is the app being "shown".